__tests__/e2e/scripts/setup_integration.sh

```diff
@@ -9,7 +9,17 @@ eval "$(elastic-package stack shellinit)"
 elastic-package stack down
 
 # start elastic-package
-env ELASTICSEARCH_IMAGE_REF=$1 ELASTIC_AGENT_IMAGE_REF=$1 KIBANA_IMAGE_REF=$1 elastic-package stack up -d -v --version $1
+env ELASTICSEARCH_IMAGE_REF=$1 ELASTIC_AGENT_IMAGE_REF=$1 KIBANA_IMAGE_REF=$1 elastic-package stack up -d -v --version $1 --services "elasticsearch"
+
+curl -X PUT "http://elastic:changeme@localhost:9200/_cluster/settings?pretty" -H 'Content-Type: application/json' -d'
+{
+    "persistent" : {
+      "cluster.routing.allocation.disk.threshold_enabled" : false
+    }
+}
+'
+
+env ELASTICSEARCH_IMAGE_REF=$1 ELASTIC_AGENT_IMAGE_REF=$1 KIBANA_IMAGE_REF=$1 elastic-package stack up -d -v --version $1 --services "elastic-agent"
```

Comment:
NIT: since we're specifying `elastic-agent` and `elasticsearch` as the only services for the `elastic-package` to start, do we even need to specify the Kibana image ref?

Reply:
Yes, you see, elastic-agent is dependent on Kibana. You can't actually start up `elastic-agent` on without Kibana. Only specifying only elastic-agent is really just shorthand for `kibana,elastic-agent,elasticsearch`as both kibana and elasticsearch are dependencies for elastic-agent.